### PR TITLE
Close #36

### LINKED ISSUE #36: ใส่ Indicator เพื่อบอกว่า ตั๋วรอบแรกขายหมดแล้ว
ตัวอย่าง:

image

### PR DESCRIPTION
**Pull request by pajaploy**
by ppllooyy_kung@hotmail.com

### CODE REVIEW

**Comment by turboza** [comment]
Please also attach the screenshot too krub

**Comment by turboza** [comment]
@pajaploy Please also attach the screenshot too krub

File: components/TicketsSection.js [hunk 1]
```diff
@@ -18,12 +18,16 @@ export default function TicketsSection () {
 function TicketsTable () {
   return (
     <div className='tickets-table'>
-      <TicketsRow title='1st round' date='18 May 2017' />
+      <TicketsRow className='soldout' title='1st round' date='18 May 2017' />
```
Comment (dtinth) [inline components/TicketsSection.js:21]: I think this does not work, as the `TicketsRow` does not take a `className` props.

For a working solution, please check out [the changes in #43](https://github.com/reactbkk/2.0.0/pull/43/files).

**Review by dtinth — COMMENTED** [review]
Hello, thanks for your participation!

However, this issue is already solved by #43, so I have to close this PR.

There are still more places that you may be able to contribute. If you can improve our site in any way, please don’t hesitate!

**Pull request closed without merging**